about.html

```diff
@@ -0,0 +1,129 @@
+          <a class="top-bar1-p" href="">Login/Register</a>
+        </nav>
+        <nav class="top-var2">
+          <div id="logo-icon">
+            <a href="./index.html"
+              ><img id="main-logo" src="./images/nimplayLogo.jpg" alt="Nimplay"
+            /></a>
+          </div>
+          <div id="main-bar2">
+            <a href="./index.html" class="text-orange top-text-var">Home</a>
+            <a href="./about.html" class="text-gray top-text-var">About</a>
+            <a id="button-main-bar" href="https://nimplay.itch.io/">Jam</a>
+          </div>
+        </nav>
+      </header>
+      <section class="home" id="hero">
+        <div class="home-titles">
+          <h2 id="hello" class="text-orange">"Hello world"</h2>
+          <h1 id="maintitle" class="text-orange">NIMPLAY jam 2023</h1>
+        </div>
+        <div class="home-p-box">
+          <p class="text-gray">
+            Indie video game company. dedicated to the development of apps and
+            games. Creating a community of developers for the creation of games,
+            jams and challenges to promote the development of Video Game
+            Industry.
+          </p>
+        </div>
+        <div class="home-button-text">
+          <p class="home-date text-gray">
+            Please contact us by Email for questions and suggestions.
+          </p>
+          <h3 class="email text-gray">nimrod7day@gmail.com</h3>
+        </div>
+      </section>
+    </div>
+    <section class="program">
+      <h2 id="program-title">
+        Nimplay<br />
+        2017
+      </h2>
+      <div class="red-line"></div>
+      <p class="home-date text-gray">
+        The logo of NIMPLAY video game development.
+      </p>
+      <div id="program-card">
+        <img src="./images/nimplayLogo.jpg" alt="" />
+      </div>
+    </section>
+    <div id="gray-line"></div>
+    <section class="program">
+      <h2 id="program-title">Previous Projects</h2>
+      <div class="red-line"></div>
+      <p class="home-date text-gray">
+        Take a look at the las two jams Projects
+      </p>
+      <div id="program-card2">
+        <div id="history">
+          <h1>2020</h1>
+          <p>Hunter Jam</p>
+        </div>
+        <div id="history2">
+          <h1>2021</h1>
+          <p>Campfire Jam</p>
+        </div>
+      </div>
+    </section>
+    <section class="participants">
+      <div id="speaker-continer">
+        <h2 id="program-title" class="text-color">Sponsors</h2>
+        <div id="red-line2"></div>
+        <div class="sponsor">
+          <img src="./images/arcadia.png" alt="" />
+          <img src="./images/kcc.png" alt="" />
+          <img src="./images/mozilla.png" alt="" />
+        </div>
+        <div class="sponsor2">
+          <img src="./images/lumen.png" alt="" />
+          <img src="./images/line-friends.png" alt="" />
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <img src="./images/cck-logo1.png" alt="" />
```

Comment:
Please add alternative text to all of your images.

<img width="839" alt="Screen Shot 2022-06-23 at 9 07 00" src="https://user-images.githubusercontent.com/82426682/175319171-87df775b-fb65-4812-9263-7eb7d2ffdab7.png">